src/Query/Result.php

```diff
@@ -10,16 +10,24 @@ class Result
     /** @var array */
     private $hits;
 
+    /** @var  array */
+    private $aggregations;
+
     /**
      * Result constructor.
      *
      * @param integer $totalHits
      * @param array $hits
      */
-    public function __construct($totalHits, $hits)
+    public function __construct(
+        int $totalHits,
+        array $hits,
+        array $aggregations = []
```

Comment:
You add this parameter but don't update the PHPDoc

Reply:
done